rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs

```diff
@@ -98,6 +98,13 @@ pub struct NeuronInfo {
     /// See the Visibility enum.
     #[prost(enumeration = "Visibility", optional, tag = "12")]
     pub visibility: Option<i32>,
+    /// The last time that following was "refreshed". There are two ways to refresh
+    /// the following of a neuron: set following, or vote directly. When this
```

Comment:
since voting can also refresh this I wonder if the name is misleading. Interestingly I was leaning more twoards "voting activity" which might have the same problem in the other direction.
I guess "governance activity" might be OK but long..

Reply:
How about `last_voting_or_following_timestamp_seconds`? Then isn't pretty hard to misinterpret.

Reply:
I think governance is too broad/abstract. There are many actions that do not cause this field to refresh.

I think last_voting_or_following is too concrete. What if we later change our minds, and add a third action that causes this to refresh? Or what if we take away one of those actions (e.g. voting) as something that can trigger a refresh?

How about voting_power_refreshed ?

Reply:
Done.